sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-row no-gutters>
+  <v-row no-gutters class="mb-n6">
```

Comment:
Can you please elaborate on why `class="mb-n6"` was needed?

Reply:
I did this change to reduce the spacing  between the title and the subtitle to match it as per the figma.

Reply:
Same from https://github.com/bcgov/SIMS/pull/1816#discussion_r1142728975

Reply:
Fixed this.